http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -268,15 +276,56 @@ class LeoPubsubMessageSubscriberSpec
         now
       )
       updatedRuntime <- clusterQuery.getClusterById(runtime.id).transaction
+      updatedRuntimeConfig <- updatedRuntime.traverse(r =>
+        RuntimeConfigQueries.getRuntimeConfig(r.runtimeConfigId).transaction
+      )
     } yield {
+      // runtime should be Stopping
       updatedRuntime shouldBe 'defined
       updatedRuntime.get.status shouldBe RuntimeStatus.Stopping
+      // machine type should not be updated yet
+      updatedRuntimeConfig shouldBe 'defined
+      updatedRuntimeConfig.get.machineType shouldBe MachineTypeName("n1-standard-4")
     }
 
     res.unsafeRunSync()
   }
 
-  it should "handle UpdateRuntimeMessage and update the runtime config in the database" in isolatedDbTest {
+  it should "handle UpdateRuntimeMessage and go through a stop-start transition" in isolatedDbTest {
```

Comment:
Is it worth having both of these two tests? It seems like the first one is checking to see that a machine-type change stops the runtime, and this one checks to see that it is eventually starting later - would just the starting test suffice, since a runtime can't be in starting status without having been stopped?

Reply:
Yeah I thought about that and I'm on the fence.. I kind of like that the `Stopping` test verifies that the machine type change has NOT been made yet.